cmd/templ/processor/processor.go

```diff
@@ -42,7 +42,7 @@ func shouldSkipDir(dir string) bool {
 }
 
 func FindTemplates(srcPath string, output chan<- string) (err error) {
-	return filepath.Walk(srcPath, func(currentPath string, info fs.FileInfo, err error) error {
+	return filepath.WalkDir(srcPath, func(currentPath string, info fs.DirEntry, err error) error {
```

Comment:
Shouldn't stricly be in this PR, but I noticed that Walk was being used, and WalkDir is more efficient.